packages/react-server/src/ReactFlightHooks.js

```diff
@@ -77,6 +77,10 @@ export const HooksDispatcher: Dispatcher = {
   useImperativeHandle: (unsupportedHook: any),
   useEffect: (unsupportedHook: any),
   useId,
+  useHostTransitionStatus: (unsupportedHook: any),
+  useOptimistic: (unsupportedHook: any),
+  useFormState: (unsupportedHook: any),
+  useActionState: (unsupportedHook: any),
```

Comment:
@sebmarkbage could you double check this? I don't think the flight dispatcher had these hooks defined before as unsupported.

Reply:
Yea, it doesn't really need it because the outer export shouldn't be available in that environment. This is just protecting against a misconfigured environment but it's good to have that backup so you know why.